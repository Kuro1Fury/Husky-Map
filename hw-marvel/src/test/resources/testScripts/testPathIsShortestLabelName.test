# Test that the path is the shortest based on label name

CreateGraph A

AddNode A n1
AddNode A n2
AddEdge A n1 n2 Edge1
AddEdge A n1 n2 Edge2

FindPath A n1 n2