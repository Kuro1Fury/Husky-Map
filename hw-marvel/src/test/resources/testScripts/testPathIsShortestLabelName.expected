# Test that the path is the shortest based on label name

created graph A

added node n1 to A
added node n2 to A
added edge Edge1 from n1 to n2 in A
added edge Edge2 from n1 to n2 in A

path from n1 to n2:
n1 to n2 via Edge1